internal/rules/resources.go

```diff
@@ -0,0 +1,123 @@
+package rules
+
+import (
+	"context"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/titpetric/factory"
+
+	"github.com/crusttech/crust/internal/auth"
+	"github.com/crusttech/crust/system/types"
+)
+
+type Access string
+
+const (
+	Allow   Access = "yes"
+	Deny           = "no"
```

Comment:
Can I suggest usage of true/false instead of yes/no?

Reply:
Not sure how this will affect the "inherit" access flag.

@titpetric already confirmed the change on db level: https://github.com/crusttech/crust/pull/20#pullrequestreview-193193169

Reply:
It does not affect inherit flag.